pom.xml

```diff
@@ -276,6 +276,12 @@
                     <version>2.8.1</version>
                     <configuration>
                         <argLine>${argLine} -Djava.awt.headless=true -noverify</argLine>
+                        <includes>
+                            <include>**/*.java</include>
```

Comment:
Hmmm I thought that we had test classes without Test on them at all (which then failed coverages checks). Will try this though and let you know :+1: Good find.